docresources/SyncBot/syncbot-quickstart.md

```diff
@@ -54,7 +54,7 @@ A little Bot, using the ProcBot framework, that links communication services.
 * A new thread in a source flow will become a new linked thread in a destination flow
 * A new message on a linked thread will become a new message in all its linked threads
 * If a service does not support privacy, then privacy is assumed by default
-* To make a comment public use `#reply`
+* Flowdock does nothing to distinguish internal from public, so to make a comment public use `%` as the very first character
```

Comment:
This seems to say that:
```
%Hello there
```
will work, but looking at the code below it's `% `?

I've missed some of this, as well. The agreement was to move to `%` from `>` because of quoting clashes?

Reply:
To answer the second of your questions first: https://www.flowdock.com/app/rulemotion/public-s-premium/threads/ZOSMP_oD-T12zm7cMUvmIiEdM6o (or, yes, it's back to the original decision)
The first question now, note the regex uses `*`, ie any amount, of literal spaces.